### PR TITLE
[C-2276] Fix upload track store updates

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Fixes issue where uploading a track does not add uploaded track to cache, or update user track_count, or update user's track lineup. This fixes the case where going to your profile after uploading a track wasnt showing the track.

### CODE REVIEW

File: packages/web/src/common/store/pages/profile/lineups/tracks/watchUploadTracksSaga.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+function* addUploadedTrackToLineup(action: UploadTracksSucceededAction) {
+  const { id, trackMetadatas } = action
+  const accountHandle = yield* select(getUserHandle)
+  if (!id || !trackMetadatas || !accountHandle) return
+
+  // We will only be adding single track uploads since multi-adds for lineups
+  // are cumbersome. When we want to suport multi-track upload on mobile or
+  // properly cache desktop profile lineups, we should revisit this.
```
Comment (Kyle-Shanks) [inline packages/web/src/common/store/pages/profile/lineups/tracks/watchUploadTracksSaga.ts:28]: 👍

**Review by Kyle-Shanks — APPROVED** [review]

**Review by raymondjacobson — APPROVED** [review]
Seems good as a near-term fix. Ideally architecturally would be handled better, do you have thoughts on how that could work?

Maybe we should just re-fetch the account and feed after the upload?

**Comment by dylanjeffers** [comment]
> Seems good as a near-term fix. Ideally architecturally would be handled better, do you have thoughts on how that could work?
> 
> Maybe we should just re-fetch the account and feed after the upload?

Im open to anything, i based this off what we do when we favorite a track, adding it to favorite lineup etc. Do we want to do something different in this case? i do like idea of simply refetching account + lineups

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-2276-fix-upload-track-store-updates

**Pull request merged**